AntennaTracker/tracking.cpp

```diff
@@ -61,7 +61,11 @@ void Tracker::update_bearing_and_distance()
     nav_status.distance = get_distance(current_loc, vehicle.location_estimate);
 
     // calculate altitude difference to vehicle using gps
-    nav_status.alt_difference_gps = (vehicle.location_estimate.alt - current_loc.alt) / 100.0f;
+    if (g.alt_source == ALT_SOURCE_GPS){
+        nav_status.alt_difference_gps = (vehicle.location_estimate.alt - current_loc.alt) / 100.0f;
+    } else if (g.alt_source == ALT_SOURCE_GPS_VEH_ONLY){
```

Comment:
we should probably remove the second if so that if the user sets the ALT_SOURCE to some unsupported number (like "4") we treat it as if they've selected "3".  I think this would be better than not updating the alt-difference-gps at all.